packages/esm-patient-chart-app/src/visit/hooks/useVisitAttributeType.tsx

```diff
@@ -62,21 +52,11 @@ export function useVisitAttributeType(uuid) {
 }
 
 export function useConceptAnswersForVisitAttributeType(conceptUuid) {
-  const { data, error } = useSWRImmutable<FetchResponse<Concept>, Error>(
+  const { data, error } = useSWRImmutable<FetchResponse<Concept>, any>(
```

Comment:
Not sure why you've changed the typing here. However, if you just want to indicate "we don't care what this is", `unknown` is more correct than `any`...

Reply:
We are using the `error` variable, it's just that I'm not type casting it